plugin/core/views.py

```diff
@@ -564,7 +564,7 @@ def format_diagnostic_for_panel(diagnostic: Diagnostic) -> Tuple[str, Optional[i
             formatted.append(code)
     lines = diagnostic.message.splitlines() or [""]
     # \u200B is the zero-width space
-    result = "{:>4}:{:<4}{:<8}{} \u200B{}".format(
+    result = "\t{:>4}:{:<4}{:<8}{} \u200B{}\n".format(
```

Comment:
The `\n` doesn't work for multi-line messages. You'd need to append the additional `\n` somewhere at the end of this function, after the additional indented lines of the message have been added.

Having said that, I prefer no empty lines :)

Reply:
Actually, it's better if the WindowManager took care of that extra `\n`, if we're going to add that. But again I prefer no `\n` :)

Reply:
I removed the `\n` and `\t` from those lines with 02d6859 
```diff
- \t{:>4}:{:<4}{:<8}{} \u200B{}\n\
+ {:>5}:{:<4}{:<8}{} \u200B{}\
```